projects/Amlogic/bootloader/canupdate.sh

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+# detect legacy kernel installs and abort to prevent upgrades
+if [ "$(uname -r)" = "3.14.29" ]; then
```

Comment:
can we print something to screen like 
```
echo "update from 3.14 is not supported"
sleep 10
```
just to reduce support overhead :)

Reply:
If the upgrade is not compatible there will already be a 60 second delay to allow the user to read whatever messages have been output during the compatibility check - I wouldn't have thought an extra 10 second delay here is necessary.